pl_examples/basic_examples/lightning_module_template.py

```diff
@@ -213,17 +213,17 @@ def __dataloader(self, train):
 
         return loader
 
-    @pl.data_loader
+    @data_loader
```

Comment:
i prefer pl.data_loader because it's clear where the decorator came from

Reply:
I like this import as it is explicit what all function you really need from particular module, but no big deal...